### ISSUE TITLE
Need some help

### ISSUE THREAD

**Issue by 1chizhang** [hanxuhfut/Decoupled-Low-light-Image-Enhancement#1]
Can you give me your training code, some details want to see again, especially your loss function, can give me a good implementation of it, thanks a lot!
